index.js

```diff
@@ -59,14 +59,58 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+
 function toggleColor(element) {
     if (element.style.color === "green") {
-        element.style.color = "black"
+        element.style.color = "red"
     } else {
         element.style.color = "green"
     }
 }
 
+header.addEventListener('click', function(){
+    if (header.style.color === 'green'){
+        header.style.color = "red"
+    } else {
+        header.style.color = "green"
+    }  
+})
+
 /***** Deliverable 2 *****/
 
+const button = document.querySelector('.like-button')
+button.addEventListener('click', function(){
+
+   const likes = document.querySelector('.likes')
+   likes.textContent = `${traveler.likes + 1} Likes `
+
+})
+
 /***** Deliverable 3 *****/
+
+const form = document.querySelector('#new-animal-sighting-form')
+form.addEventListener('submit', function(event){
+    event.preventDefault()
+
+    const speciesInput = event.target.species.value
+    const linkInput = event.target.link.value
+    const photoInput = event.target.photo.value
+    const descriptionInput = event.target.description.value
+
+    
+    const lastIndex = traveler.animalSightings.length -1
+    const lastId = traveler.animalSightings[lastIndex].id
+
+    newAnimalSighObj = {
+        travelerId: traveler.id,
+        species: speciesInput,
+        photo: photoInput,
+        link: linkInput,
+        description: descriptionInput,
+        id: lastId + 1
+    }
+    renderAnimalSightingPost(newAnimalSighObj)
+})
+
```

Comment:
Great work on this last one!! If you wanted to get fancy w/ the ES6 syntax, you could make sure your variable names were the same names as the keys and do something like this...

```javascript
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species,
        videoLink,
        photo,
        description
    }
```
But I like your way and it's very readable and makes sense! Just showing you in case you see it done other ways.

PPS, what if we wanted to clear the form on a submit? There's probably a function on the event.target object that lets us do that... 🤔